Dockerfile.recommender

```diff
@@ -1,5 +1,5 @@
 # Stage 1: build the monorepo
-ARG version=1.0.0
+ARG version=1.1.0
```

Comment:
@raymondfeng , I don't see the point of bumping up the numbers from 1.0.0.  
This is something the user will have to do in order to deploy any changes to the cloud (so cloud doesn't use an old cached image). If you set it to 1.1.0. And I deploy to cloud and make a change... then I need to bump the numbers anyways.

Reply:
The version bump is for the project itself as we're adding new features to the repo, which deserve a new minor version.